frontend/src/scenes/data-warehouse/settings/DataWarehouseSourcesTable.tsx

```diff
@@ -0,0 +1,325 @@
+import { TZLabel } from '@posthog/apps-common'
+import { LemonButton, LemonDialog, LemonSwitch, LemonTable, LemonTag, Link, Spinner, Tooltip } from '@posthog/lemon-ui'
+import { useActions, useValues } from 'kea'
+import { router } from 'kea-router'
+import { ProductIntroduction } from 'lib/components/ProductIntroduction/ProductIntroduction'
+import { More } from 'lib/lemon-ui/LemonButton/More'
+import { urls } from 'scenes/urls'
+
+import { DataTableNode, NodeKind } from '~/queries/schema'
+import { ExternalDataSourceSchema, ExternalDataStripeSource, ProductKey } from '~/types'
+
+import { dataWarehouseSettingsLogic } from './dataWarehouseSettingsLogic'
+
+const StatusTagSetting = {
+    Running: 'primary',
+    Completed: 'success',
+    Error: 'danger',
+    Failed: 'danger',
+}
+
+export function DataWarehouseSourcesTable(): JSX.Element {
```

Comment:
Longer term we'll want to include enabled only imports in the overview page, this can be done now if FF is true (separate PR maybe)